pkg/controller/controller.go

```diff
@@ -218,20 +218,26 @@ func (c *Controller) onUpdateActionSet(oldAS, newAS *crv1alpha1.ActionSet) error
 		log.Print("Updated ActionSet", field.M{"ActionSetName": newAS.Name})
 		return err
 	}
+	ctx := context.TODO()
+	ctx = field.Context(ctx, consts.ActionsetNameKey, newAS.GetName())
+	// adding labels in the context as field for better logging
+	for key, value := range newAS.GetLabels() {
```

Comment:
What are the benefits of logging the actionset labels? Since labels are something that user can change over time, will they actually want to aggregate these fields?